newsroom/wire/block_media/download_items.py

```diff
@@ -0,0 +1,88 @@
+from newsroom.auth import get_user
+from newsroom.wire.block_media.company_factory import CompanyFactory
+from newsroom.wire.block_media.filter_media import get_allowed_tags
+
+from lxml import html as lxml_html
+import re
+import logging
+logger = logging.getLogger(__name__)
+
+
+def filter_items_download(func):
+    def wrapper(_ids, item_type, filter_func=None):
+        items = func(_ids, item_type)
+        if filter_func and items and (item_type != 'agenda'):
+            items = filter_func(items)
+        return items
+    return wrapper
+
+
+def block_items_by_embedded_data(items):
```

Comment:
docstrings would be beneficial for non-simple methods, so it explains to other developers what this method does